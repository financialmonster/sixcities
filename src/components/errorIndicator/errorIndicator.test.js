import React from 'react';
import renderer from 'react-test-renderer';

import ErrorIndicator from './errorIndicator.jsx';

it(`Correctly renders the ErrorIndicator component`, () => {
    const tree = renderer.create(<ErrorIndicator operation="loading of the list of reviews" />);

    expect(tree).toMatchSnapshot();
});
