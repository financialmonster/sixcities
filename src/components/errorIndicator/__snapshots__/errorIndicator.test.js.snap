// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Correctly renders the ErrorIndicator component 1`] = `
<div
  className="cities__status-wrapper tabs__content"
>
  <b
    className="cities__status"
  >
    We're sorry
  </b>
  <p
    className="cities__status-description"
  >
    It's been an error during the process of 
    loading of the list of reviews
    . Please, reload the page and try again
  </p>
</div>
`;
